src/Uno.UWP/ApplicationModel/DataTransfer/DataPackage.Android.cs

```diff
@@ -0,0 +1,78 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+	public partial class DataPackage
+	{
+		internal string _html;
+		internal Uri _uri;
+		public void SetText(string text)
+		{
+			if (text == null)
+			{
+				throw new ArgumentNullException("Cannot set DataPackage.Text to null");
+			}
+
+			this.Text = text;
+		}
+
+		// Uri with namespace, as we have two different Uri (UWP and Android) here
+		public void SetUri(global::System.Uri value)

```

Comment:
```suggestion
		public void SetUriUri value)

```

Use `using` instead.

Reply:
See my comment above this method: "Uri with namespace, as we have two different Uri (UWP and Android) here".
First, compiler has to be sure which Uri it should use in which line.
Second, any (human) reader of this code should know which Uri is used where, without much effort to fully understand which, were and why.

Reply:
You can use an using alias, `using Uri = System.Uri;`, to avoid collision.
And, refer to it as `Uri` below.

Reply:
The _new_ convention on Uno is to prefix such namespace alias with `_`, giving `using _Uri = System.Uri;`

Reply:
But these methods are, converters between two different Uri; in short:
`Uri.Parse(Uri.ToString())` and `Uri(Uri.ToString())` 
I think... no, I'm sure!, that `Android.Net.Uri.Parse(System.Uri.ToString())` and `System.Uri(Android.Net.Uri.ToString())` are much more readable, intention is more clear, and making error (while editing code) is much less probably.
So, I refuse to make code more errorprone.